dropbox_rclone/dropbox_rclone/scripts/cs_dropbox_download.py

```diff
@@ -5,8 +5,9 @@
 
 import tarfile
 
+from dropbox_rclone import contants as const
 
-# rclone ls dropbox:Cybershake/v22p12|grep -e "Source\|IM"| sed -E 's/^[[:space:]]+//' |cut -d" " -f2 |xargs -I {} rclone copy dropbox:Cybershake/v22p12/{} . --progress
+# rclone ls dropbox:/Quake\ CoRE/Public/Cybershake/v22p12|grep -e "Source\|IM"| sed -E 's/^[[:space:]]+//' |cut -d" " -f2 |xargs -I {} rclone copy dropbox:/Quake\ CoRE/Cybershake/v22p12/{} . --progress
```

Comment:
Is this comment even useful?

Reply:
> Is this comment even useful?

haha, good point. I must have fallen in love with this complex command during the development. Removed  :)